shared/@common/api/userAPI.ts

```diff
@@ -1,17 +1,14 @@
 import { axiosInstance } from '@/shared/utils/axiosInstance';
 
 const userAPI = {
-  getUserData: async (user_id: string) => {
-    const data = await axiosInstance.get(`/users/${user_id}`);
-    return data.data;
+  getUserData: (user_id: string) => {
+    return axiosInstance.get(`/users/${user_id}`);
   },
-  post: async <T>(user_id: T, body: T) => {
-    const data = await axiosInstance.post(`/users/${user_id}`, body);
-    return data.data;
+  post: <T>(user_id: T, body: T) => {
+    return axiosInstance.post(`/users/${user_id}`, body);
   },
-  put: async <T>(user_id: T, body: T) => {
-    const data = await axiosInstance.put(`/users/${user_id}`, body);
-    return data.data;
+  put: <T>(user_id: T, body: T) => {
+    return axiosInstance.put(`/users/${user_id}`, body);
```

Comment:
제가 타입스크립트에 많이 미숙합니다.
현재 제네릭을 `user_id`에도 넣어주셨는데 이유가 따로 있을까요❓
다른 API 함수에도 똑같은 방식으로 들어가 있더라구요
저는 이렇게 생각했습니다!
```suggestion
  put: <T>(user_id: string, body: T) => {
    return axiosInstance.put(`/users/${user_id}`, body);
```

Reply:
한 번에 수정하다보니 잘못 넣었네요 수정할게요 알려주셔서 감사합니다☺️